test-e2e/panels/spectra.test.ts

```diff
@@ -89,7 +89,7 @@ test('Check change spectrum color, Should be white', async ({ page }) => {
   await expect(whiteSpectrumLine).toBeVisible();
 });
 
-test('Should 2d deactivate spectrum', async ({ page }) => {
+test.skip('Should 2d deactivate spectrum', async ({ page }) => {
```

Comment:
You forgot to remove the '.skip'. There is also the possibility to focus on a test using '.only'. Could be easier than adding skip everywhere.

Reply:
yes sorry, thank youu